check-symbiosis.lic

```diff
@@ -0,0 +1,83 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#check-symbiosis
+=end
+
+custom_require.call(%w[common common-arcana drinfomon spellmonitor])
+
+class SymbiosisCheck
+
+  def initialize
+    arg_definitions = [
+      [
+        {
+          name: 'symbiosis_type',
+          optional: true,
+          options: %w[activate avoid cast discern endure examine explore harness harvest heal impress learn perform remember resolve spell spring strengthen watch],
+          description: "Which symbiosis do you want to check? Argument -> Yaml -> defaults to harness"
+        }
+      ]
+    ]
+    args = DRC.parse_args(arg_definitions)
+
+    symbiosis_type = args.symbiosis_type || get_settings.symbiotic_research_type

```

Comment:
Just nitpicking but you could downcase it here as a sort of input processing stage. Then you don't have to do it downstream.